Traits/CanOwnTrait.php

```diff
@@ -5,67 +5,64 @@
 use Apiato\Core\Exceptions\CoreInternalErrorException;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Support\Str;
-use Throwable;
 
 trait CanOwnTrait
 {
     /**
-     * Checks if the model is the owner of the $ownable model
-     * by comparing IDs
+     * Checks if the model is the owned by the $owner model.
```

Comment:
minus 'the'

Reply:
Like this?
`Checks if model is the owned by the $owner model.`

Reply:
Checks if the model is owned by the $owner model.